index.html

```diff
@@ -135,6 +137,9 @@ <h3>Latest news about <span>[populate name/industry/sector from the input form]<
         </section>
     </main>
 
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="articles.js"></script>
```

Comment:
Generally, as I understand, the suggestion is to call the scripts after all the HTML code - the last thing to do before </body> tag. If you move this link below, just below the jQuery link but before my own code I've included, I think that should work best. Eventually, I'll move my code to the .js file as well - once we figure out the full sequence of calls and functions.